### DISCUSSION TITLE
Download Bruno to test our Backend API

### DISCUSSION THREAD

**Discussion by umitcan07** [bounswe/bounswe2024group11#265] [Announcements]
Let's download [Bruno](https://www.usebruno.com/downloads) , open source API Client to test our endpoints.
You will be able to send requests and see response within different pre-defined environments without needing a frontend, which will speed up our test workflow.
Additionally, our endpoints in Bruno will be synced.

**Comment by MucahitErdoganUnlu** [comment]
Installed it. Very nice environment! [bruno guide](https://github.com/bounswe/bounswe2024group11/wiki/Using-Bruno-to-test-API)

**Comment by Meminseeker** [comment]
I've added configuration files to `/bruno` directory. You can open that directory in your Bruno app and start using it 👌
